tests/testthat/test.dbDeleteRowByKey.R

```diff
@@ -30,5 +30,12 @@ test_that("Exactly 50000 rows are deleted", {
 
 })
 
+test_that("The function will safly exit if given empty dt", {
+
+  # Expect there is no error
+  expect_error(dbDeleteRowByKey(db, "Letters", dt_letters[1 == 0]), NA)
```

Comment:
Yes, really stupid but the NA at the end of the line means that it is expecting the error that is NA, which is no error.